plugins/metadata/plugin.go

```diff
@@ -44,6 +44,54 @@ func init() {
 	timeout.Set(boltOpenTimeout, 0) // set to 0 means to wait indefinitely for bolt.Open
 }
 
+// BoltOptions replicates bolt.Options struct, but excludes function fields and adds tags to allow TOML marshalling.
+type BoltOptions struct {
+	// Sets the DB.NoGrowSync flag before memory mapping the file.
+	NoGrowSync bool `toml:"no_grow_sync"`
```

Comment:
Can we update the comments to follow godoc conventions, including the symbol name as the first word?

Reply:
Yup, fixed.